src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final int max = 100;
+
+    public Ball getRandomBall() {
+        ColorSupplier color = new ColorSupplier();
+        Random random = new Random();
```

Comment:
the same as ColorSupplier

Reply:
Could you please elaborate, I don't understand